x-pack/plugins/case/server/client/configure/get_fields.ts

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
nit -> I think is better to rename from `x-pack/plugins/case/server/client/configure/get_fields.ts` to `x-pack/plugins/case/server/client/connectors/get_fields.ts`. `configure` -> `connectors`

Reply:
I'm following the paths established in `x-pack/plugins/case/server/routes/api/cases`. There is not a connectors directory, but the calls to `get_connectors` and `get_fields` exist in the `configure` directory. thats why i did this